ros_docker_images/templates/docker_images/create_base_image.Dockerfile.em

```diff
@@ -26,10 +19,16 @@ ENV TZ @timezone
 
 # install bootstrap tools
 RUN apt-get update && apt-get install -q -y \
+    locales \
     python-rosdep \
     python-rosinstall \
     python-vcstools
 
+# setup environment
+RUN locale-gen en_US.UTF-8
+ENV LANG en_US.UTF-8
+ENV TZ @timezone
```

Comment:
I think the official library maintainers may question this, never seen it used elsewhere. Also, LC_ALL is not needed? I've seen it listed in the ROS2 install, but not sure we've needed it before.

https://unix.stackexchange.com/questions/87745/what-does-lc-all-c-do